plugins/modules/elb_target_group.py

```diff
@@ -897,6 +914,9 @@ def main():
     connection = module.client('elbv2', retry_decorator=AWSRetry.jittered_backoff(retries=10))
 
     if module.params.get('state') == 'present':
+        if module.params.get('protocol') in ['http', 'https', 'HTTP', 'HTTPS'] and module.params.get('deregistration_connection_termination', None):
```

Comment:
```suggestion
        if module.params.get('protocol').upper() in ['HTTP', 'HTTPS'] and module.params.get('deregistration_connection_termination', None):
```

Reply:
Basically I'm fine with both. But it's already used like this: https://github.com/ansible-collections/community.aws/blob/main/plugins/modules/elb_target_group.py#L526